cluster/clusterpb/cluster.pb.go

```diff
@@ -579,6 +579,6 @@ func skipCluster(dAtA []byte) (n int, err error) {
 }
 
 var (
-	ErrInvalidLengthCluster = fmt.Errorf("proto: negative length found during unmarshaling")
+	ErrInvalidLengthCluster = fmt.Errorf("proto: negative length found during unmarshalling")
```

Comment:
This repository is split ~60:40 in favor of doubled `l` over single `l`. There's one API which uses single `l`, and thus that can't be changed.

I can of course drop any commit, so if you want to be inconsistent, that's fine....

Reply:
Dropped because this is actually talking to a third party module.